pyrasterframes/src/main/python/setup.py

```diff
@@ -47,6 +47,16 @@ def _divided(msg):
     return divider + '\n' + msg + '\n' + divider
 
 
+from pweave import PwebPandocFormatter
```

Comment:
would prefer this is at the top of the file... This seems to be causing the travis woes but not sure why

Reply:
after getting some rest I now realize the problem is you cannot `import` this at this point in the script. You should consider  breaking this `PegdownMarkdownFormatter` class out into another script that you import within the PweaveDocs class. Perhaps you can define this class within `PweaveDocs`